### PR TITLE
Add databricks_automl to the conda env for arima and deepar

### PR DESCRIPTION
**Pull request by sabhya-db**
It was already passed in for Prophet. Adding for ARIMA and DeepAR, too, so we can log correctly.

### CODE REVIEW

**Review by apeforest — COMMENTED** [review]
Can you add a unit test

**Comment by sabhya-db** [comment]
> Can you add a unit test

@apeforest the existing UT to log, load, and predict an ARIMA model passes, see here: https://github.com/databricks/automl/blob/branch-0.2.20.3/runtime/tests/automl_runtime/forecast/pmdarima/model_test.py#L432

I could add a test / check to see that the requirements.txt that are logged with "automl_runtime=0.2.20.x" for all 3 models?

File: runtime/databricks/automl_runtime/forecast/model.py [hunk 1]
```diff
@@ -57,10 +57,14 @@ def mlflow_forecast_log_model(forecast_model: ForecastModel,
     :param forecast_model: Forecast model wrapper
     :param sample_input: sample input Dataframes for model inference
     """
-    # log the model without signature if infer_signature is failed.
+    # TODO: we should not be logging without a signature since it cannot be registered to UC then
```
Comment (apeforest) [inline runtime/databricks/automl_runtime/forecast/model.py]: Please add a JIRA ticket number to TODO:

e.g. # TODO(ML-XXXXX): we should not be logging...

File: runtime/tests/automl_runtime/forecast/pmdarima/model_test.py [hunk 2]
```diff
@@ -25,8 +25,10 @@
 from mlflow.protos.databricks_pb2 import ErrorCode, INVALID_PARAMETER_VALUE
 from pmdarima.arima import ARIMA
 
-from databricks.automl_runtime.forecast.pmdarima.model import ArimaModel, MultiSeriesArimaModel, AbstractArimaModel, \
-    mlflow_arima_log_model
+from databricks.automl_runtime.forecast.pmdarima.model import (
```
Comment (apeforest) [inline runtime/tests/automl_runtime/forecast/pmdarima/model_test.py:28]: please break imports into multiple lines: 

See https://peps.python.org/pep-0008/#imports

File: runtime/tests/automl_runtime/forecast/deepar/model_test.py [hunk 3]
```diff
@@ -145,10 +149,23 @@ def test_model_save_and_load_multi_series(self):
...
+
+        # load the model and predict
+        loaded_model = mlflow.pyfunc.load_model(f"runs:/{run_id}/model")
         pred_df = loaded_model.predict(sample_input)
 
         assert pred_df.columns.tolist() == [time_col, "yhat", id_col]
         assert len(pred_df) == self.prediction_length * 2
         assert pred_df[time_col].min() > sample_input[time_col].max()
+
+    def _check_requirements(self, run_id: str):
```
Comment (apeforest) [inline runtime/tests/automl_runtime/forecast/deepar/model_test.py:166]: nit: if a method is being tested in a TestClass, make it a public method (i.e. without prefix _)

**Review by apeforest — COMMENTED** [review]
Thanks @sabhya-db. A few nits. Otherwise, LGTM Thanks

**Comment by sabhya-db** [comment]
> Thanks @sabhya-db. A few nits. Otherwise, LGTM Thanks

Thanks for checking, will fix nits soon

File: runtime/tests/automl_runtime/forecast/deepar/model_test.py (hunk 3 above)
Comment (sabhya-db) [reply runtime/tests/automl_runtime/forecast/deepar/model_test.py:166]: It's not a unit test itself, it's a helper method called only within other unit test cases, should I still remove prefix?

**Review by es94129 — APPROVED** [review]
LGTM. Do you know if it's a serverless-only issue, or should we also port this to the main branch?

**Comment by sabhya-db** [comment]
> LGTM. Do you know if it's a serverless-only issue, or should we also port this to the main branch?

@es94129 

We should do this on the main branch too, an ARIMA model logged from the main branch won't be able to be served at present. For a quick hotfix you can just add "automl_runtime=xyz" to ARIMA_CONDA_ENV here: sabhya-db/ML-45532-sample-weight-2

**Comment by es94129** [comment]
Can you also port the changes for ARIMA to the main branch then?

**Comment by sabhya-db** [comment]
> Can you also port the changes for ARIMA to the main branch then?

will do in a separate PR

**Pull request merged**